src/AWS.Deploy.CLI/ServerMode/Models/SystemCapabilitySummary.cs

```diff
@@ -5,17 +5,15 @@ namespace AWS.Deploy.CLI.ServerMode.Models
 {
     public class SystemCapabilitySummary
     {
-        public string Name { get; set; }
-        public bool Installed { get; set; }
```

Comment:
The interface was changed, so the Rest API generator needs to be re-run

Reply:
Re-ran the generator and updated the PR.